conduct.md

```diff
@@ -53,11 +53,16 @@ Allegheny's honor code for all submitted work.
 
 * Acceptable Conduct (A):
   * Members shall
-    * :heavy_exclamation_mark: TODO: add items in the A category for deadlines.
+    * Turn assignments in on the due date, by the time set by JJ; the only exception to this rule is if the student is excused by the professor.
+    * Keep track of the due dates of the assignments.
+    * Be able to give reasons for missing deadlines.
 
 * Unacceptable Conduct (U):
   * Members shall not
-    * :heavy_exclamation_mark: TODO: add items in the U category for deadlines.
+    * Turn in work past the due date given.
+    * Turn in work past the time set for turn-in.
```

Comment:
These two points have a similar meaning. You can either clarify the difference between them or combine them together

Reply:
We combined them to make it more concise.